pkg/api/login_oauth.go

```diff
@@ -97,9 +97,8 @@ func OAuthLogin(ctx *middleware.Context) {
 
 		tr := &http.Transport{
 			TLSClientConfig: &tls.Config{
-				InsecureSkipVerify: true,
-				Certificates:       []tls.Certificate{cert},
-				RootCAs:            caCertPool,
+				Certificates: []tls.Certificate{cert},
```

Comment:
InsecureSkipVerify could be added to the Oauth implementation like TlsClientCert currently works. For some reason its not documented or describe in the config files... but you can find the implementation here: https://github.com/grafana/grafana/blob/master/pkg/social/social.go#L54